lib/api/src/rest/conversions.rs

```diff
@@ -7,8 +7,25 @@ use super::{
     FacetRequestInternal, FacetResponse, FacetValue, FacetValueHit, NearestQuery, OrderByInterface,
     Query, QueryInterface, VectorOutput, VectorStructOutput,
 };
+use crate::grpc::{InferenceUsage as GrpcInferenceUsage, ModelUsage};
```

Comment:
nit: for consistency, I'd rather use namespace alias, as it is less verbose when dealing with multiple types. This way it is not necessary to add one alias per type
```rust
use crate::grpc as grpc;

impl From<InferenceUsage> for grpc::InferenceUsage {
...
```